modules/zemantaBidAdapter.md

```diff
@@ -0,0 +1,95 @@
+# Overview
+
+```
+Module Name: Zemanta Adapter
+Module Type: Bidder Adapter
+Maintainer: prog-ops-team@outbrain.com
+```
+
+# Description
+
+Module that connects to zemanta bidder to fetch bids.
+Both native and display formats are supported but not at the same time. Using OpenRTB standard.
+
+# Configuration
+
+## Bidder and usersync URLs
+
+The Zemanta adapter does not work without setting the correct bidder and usersync URLs.
+You will receive the URLs when contacting us.
+
+```
+pbjs.setConfig({
+    zemanta: {
+      bidderUrl: 'https://bidder-url.com',
+      usersyncUrl: 'https://usersync-url.com'
+    }
+});
+```
+
+
+# Test Native Parameters
+```
+    var adUnits = [
+        code: '/19968336/prebid_native_example_1',
+        mediaTypes: {
+            native: {
+                image: {
+                    required: false,
+                    sizes: [100, 50]
+                },
+                title: {
+                    required: false,
+                    len: 140
+                },
+                sponsoredBy: {
+                    required: false
+                },
+                clickUrl: {
+                    required: false
+                },
+                body: {
+                    required: false
+                },
+                icon: {
+                    required: false,
+                    sizes: [50, 50]
+                }
+            }
+        },
+        bids: [{
+            bidder: 'zemanta',
+            params: {
+                publisher: {
+                  id: '2706', // required
+                  name: 'Publishers Name',
+                  domain: 'publisher.com'
+                },
+                tagid: 'tag-id'
+            }
+        }]
+    ];
+```
+
+# Test Display Parameters
+```
+    var adUnits = [
+        code: '/19968336/prebid_display_example_1',
+        mediaTypes: {
+          banner: {
+            sizes: [[300, 250]]
+          }
+        },
+        bids: [{
+            bidder: 'zemanta',
+            params: {
+                publisher: {
+                  id: '2706', // required
+                  name: 'Publishers Name',
+                  domain: 'publisher.com'
+                }
```

Comment:
missing a "," on this line.

Also, we will need you to include a test endpoint url in this markdown for the **bidderUrl** so we can confirm bids are received and respond back to prebid properly through your adapter. The latter part is missing right now.

Reply:
Ok, we will prepare and include the test url.